scripts/zephyr_module.py

```diff
@@ -97,6 +97,9 @@
       - type: str
 '''
 
+# Does not include the extension, since that can be .yml or .yaml
+MODULE_YML_PATH = PurePath('zephyr/module')
```

Comment:
when you are using `.with_suffix('.yml')` you can actually keep the suffix here for readability (and indirectly provide a preference), as the `.with_suffix()` will substitute the suffix.

Like this:
```
MODULE_YML_PATH = PurePath('zephyr/module.yml')
```

and then later:
```
 for module_yml in [module_path / MODULE_YML_PATH,
                       module_path / MODULE_YML_PATH.with_suffix('.yaml')]:
```
which will substitute `.yml` with `.yaml`.

Ref: https://docs.python.org/3/library/pathlib.html
> Return a new path with the [suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) changed. If the original path doesn’t have a suffix, the new suffix is appended instead. If the suffix is an empty string, the original suffix is removed:
> ```
> >> p = PureWindowsPath('c:/Downloads/pathlib.tar.gz')
> >> p.with_suffix('.bz2')
> PureWindowsPath('c:/Downloads/pathlib.tar.bz2')
> ```

Reply:
Very nice suggestion, implemented. Thanks!